### PR TITLE
Add a script to download naco county-level intervention data

### LINKED ISSUE #8: Consume NACo intervention data.
Write a script that pulls from https://ce.naco.org/python/data?dbset=COVID_Table&dbind=County_Declaration_Type3|County_Emergency_Declaration|County_Shelter_In_Place_Policy|County_Business_Closure_Policy|Population_total|Population_over_65_pct&dbyear=2020

and spits out a county_interventions.json file similar to our existing state one (currently here: https://github.com/covid-projections/covid-projections/blob/develop/src/assets/data/interventions.json but will be moving into this repo, see #5)

Their data looks like this:

```
{
    "columns": [
        "FIPS",
        "County_Declaration_Type3",
        "County_Emergency_Declaration",
        "County_Shelter_In_Place_Policy",
        "County_Business_Closure_Policy",
        "Population_total",
        "Population_over_65_pct"
    ],
    "data": [
        [
            "02020",
            "Both Policies",
            "Declared March 16, <a target=\"_blank\" href=\"https://www.muni.org/covid-19/Pages/default.aspx\" style=\"color: #92D0F9\">Link</a>",
            "Declared March 20, <a target=\"_blank\" href=\"https://www.muni.org/departments/mayor/pressreleases/documents/eo-03.pdf\" style=\"color: #92D0F9\">Link</a>",
            null,
            291538.0,
            0.1109049246
        ],
        ...
    ]
}
```

A non-null County_Shelter_In_Place_Policy should map to "shelter_in_place" in the resulting interventions file.

I am thinking a non-null County_Emergency_Declaration or County_Business_Closure_Policy should map to "social_distancing" but we can revisit / tweak this. So we would output:
```
{
    "02020": "social_distancing",
    ...
}
```

I think the downloader should probably live in scripts/update-interventions-naco.py and output to data/interventions-naco/

### PR DESCRIPTION
**Pull request by gsoltis**
Fixes #8 

Downloads naco county data, transforms into interventions.json format

### CODE REVIEW

**Comment by gsoltis** [comment]
Do we want to include some sort of timestamp in the output?

**Comment by mikelehen** [comment]
Yeah, I've been leaning towards creating version.txt files that contain a timestamp of some kind.

**Comment by gsoltis** [comment]
Ok, i'll add that. I'll just write out UTC time I guess, unless you know of some version for the naco data? I don't see it in the response.

File: scripts/update-interventions-naco.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+def transform_json(columns, data):
+  fips_index = columns.index('FIPS')
+  shelter_index = columns.index('County_Shelter_In_Place_Policy')
+  county_emergency = columns.index('County_Emergency_Declaration')
+  county_closure = columns.index('County_Business_Closure_Policy')
+
+  def row_intervention(row) -> str:
+    if row[shelter_index] != None:
+      return 'shelter_in_place'
+    elif row[county_emergency]:
```
Comment (mikelehen) [inline scripts/update-interventions-naco.py]: The above check has `!= None` but these don't.  I don't know python well enough to know if they're equivalent or there's something subtle going on here?

**Comment by gsoltis** [comment]
Added a `version.txt` as part of the output. Includes the UTC time that the script was run.

File: scripts/update-interventions-naco.py (hunk 1 above)
Comment (gsoltis) [reply scripts/update-interventions-naco.py]: Added them. Technically not required, but consistency and readability are probably good reasons to have them.

**Pull request merged**